.github/workflows/WorkitemValidation.yaml

```diff
@@ -15,16 +15,46 @@ defaults:
 
 jobs:
   WorkItemValidation:
+    name: 'Validate link to issues'
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v3
 
+      - name: Link work items to pull request if possible
```

Comment:
This step seems better suited for _WorkitemValidationForMicrosoft_ job. 

If it manages to link the PR to a work item, then everything will be nice and green.